common-template/src/main/resources/templates/email/Common/insightsEmailBodyLightV2.html

```diff
@@ -0,0 +1,208 @@
+{@String title-right-part-link-style = "color: black; text-decoration-line: none; text-decoration: none;"}
+{@String body-section-p-style = "margin-top: 0px; font-size: 16px; font-weight: 400;"}
+{@String body-section-underline-link-style = "text-decoration-line: underline !important; color: rgb(0, 102, 204); text-decoration-color: rgb(0, 102, 204);"}
+{@String body-section-table-style = "width: 100% !important; border-collapse: collapse !important;"}
+{@String body-section-table-th-style = "font-size: 12px; font-weight: 700; text-align: left; padding-bottom: 10px; padding-left: 10px; padding-right: 10px; padding-top: 10px;"}
+{@String body-section-table-tbody-style = "font-size: 14px; font-weight: 400;"}
+{@String body-section-table-tr-style = "border-bottom-color: rgb(184, 187, 190); border-bottom-width: 1px; border-bottom-style: solid;"}
+{@String body-section-table-td-style = "padding-bottom: 8px; padding-left: 10px; padding-right: 10px; padding-top: 8px; border-top-color: rgb(184, 187, 190); border-top-width: 1px; border-top-style: solid; line-height: 22px;"}
+{@String body-section-table-td-a-style = "color: rgb(0, 102, 204); text-decoration-color: rgb(0, 102, 204); text-decoration-line: none; text-decoration: none;"}
+{@String body-badge-style = "font-size: 16px; font-family: 'Red Hat Text', sans-serif; font-weight: 400; display: inline-block; padding-right: 0.5rem; padding-left: 0.5rem; text-align: center; background-color: #f0f0f0; border-radius: 30em;"}
+
+{@String query_params = "from=notifications&integration=daily_digest"}
+    {#if renderCustomLinks.or(false)}
```

Comment:
The indentation from this line until the end of the file makes it look like there's a parent. Maybe move everything 4 chars left?